projects/refer-and-monitor-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/api/resource/ProbationCaseResource.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.hmpps.api.resource
+
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.api.model.ResponsibleOfficer
+import uk.gov.justice.digital.hmpps.service.ManagerService
+
+@RestController
+@RequestMapping("probation-case/{crn}")
+class ProbationCaseResource(private val managerService: ManagerService) {
+    @PreAuthorize("hasRole('CRS_RESPONSIBLE_OFFICER')")
```

Comment:
Do we need a separate role for this?  Will it always be called by the same client (refer-and-monitor)?